auth/webRoutes.py

```diff
@@ -378,6 +386,63 @@ def getGroupUsers(group):
         return make_response(json.dumps({"users": usersSafe}), 200)
 
 
+# passwd related endpoints
+@app.route('/passwd/reset/<username>', methods=['POST'])
+def passwdResetRequest(username):
+    if conf.emailHost == 'NOEMAIL':
+        return formatResponse(501, "Feature not configurated")
```

Comment:
"Feature not ~[configurated](https://www.merriam-webster.com/dictionary/configurated)~ configured"